modules/apps/27-interchain-accounts/controller/ibc_middleware.go

```diff
@@ -101,7 +103,10 @@ func (im IBCMiddleware) OnChanOpenAck(
 	}
 
 	// call underlying app's OnChanOpenAck callback with the counterparty app version.
-	return im.app.OnChanOpenAck(ctx, portID, channelID, counterpartyChannelID, counterpartyVersion)
+	if im.app != nil {
+		return im.app.OnChanOpenAck(ctx, portID, channelID, counterpartyChannelID, counterpartyVersion)
+	}
+	return nil
 }
```

Comment:
```suggestion
	if im.app != nil {
		return im.app.OnChanOpenAck(ctx, portID, channelID, counterpartyChannelID, counterpartyVersion)
	}
	
	return nil
}
```
nit: I think it is nice just to space these out